MerlinAU.sh

```diff
@@ -496,17 +496,18 @@ readonly FW_FileName="${PRODUCT_ID}_firmware"
 readonly FW_URL_RELEASE="${FW_URL_BASE}/${PRODUCT_ID}/${FW_URL_RELEASE_SUFFIX}/"
 
 ##------------------------------------------##
-## Modified by ExtremeFiretop [2024-Feb-01] ##
+## Modified by ExtremeFiretop [2024-May-21] ##
 ##------------------------------------------##
 logo() {
   echo -e "${YLWct}"
-  echo -e "    __  __           _ _               _    _ "
-  echo -e "   |  \/  |         | (_)         /\  | |  | |"
-  echo -e "   | \  / | ___ _ __| |_ _ __    /  \ | |  | |"
-  echo -e "   | |\/| |/ _ | '__| | | '_ \  / /\ \| |  | |"
-  echo -e "   | |  | |  __| |  | | | | | |/ ____ | |__| |"
-  echo -e "   |_|  |_|\___|_|  |_|_|_| |_/_/    \_\____/ ${GRNct}v${SCRIPT_VERSION}"
+  echo -e "      __  __           _ _               _    _ "
+  echo -e "     |  \/  |         | (_)         /\  | |  | |"
+  echo -e "     | \  / | ___ _ __| |_ _ __    /  \ | |  | |"
+  echo -e "     | |\/| |/ _ | '__| | | '_ \  / /\ \| |  | |"
+  echo -e "     | |  | |  __| |  | | | | | |/ ____ | |__| |"
+  echo -e "     |_|  |_|\___|_|  |_|_|_| |_/_/    \_\____/ ${GRNct}v${SCRIPT_VERSION}"
   echo -e "                                              ${NOct}"
+  printf "${YLWct}============ By ExtremeFiretop & Martinski W. ============${NOct}\n\n"
```

Comment:
Added our names to the logo, I thought it already was but this makes it more consistent across the pages.

Reply:
> Added our names to the logo, I thought it already was but this makes it more consistent across the pages.
>
Honestly, I like the way it was before with our name handles showing only on the top Main Menu. Putting our names on every sub-menu feels rather unnecessary and it seems a bit like "stroking our own egos." In any case, it's a very trivial detail so I won't comment any further about it.

Reply:
Okay fair enough, I can change this too if you dislike it, I honestly always intended for our names to be apart of the logo it threw me off when I noticed it wasn't and when I added it it felt more consistent but you think it makes us look ego searching I can just move it back that's no problem.

This is probably what caused the confusion when I said "tiny header" you may have thought I meant this signature handle because it wasn't apart of the logo but in my head it was that's why I did the change

Reply:
> Okay fair enough, I can change this too if you dislike it, I honestly always intended for our names to be apart of the logo it threw me off when I noticed it wasn't and when I added it it felt more consistent but you think it makes us look ego searching I can just move it back that's no problem.
> 
It's just my personal take on it. Technically, by next week I likely won't even notice or care that it's there in the sub-menus. And again, it's such a trivial detail that it's not worth spending more time & effort on it.

>
> This is probably what caused the confusion when I said "tiny header" you may have thought I meant this signature handle because it wasn't apart of the logo but in my head it was that's why I did the change
>
Yes, the signature handle (which was not part of the logo function at the time) is what came to my mind when I read "tiny header" in your reply.

Reply:
> > Okay fair enough, I can change this too if you dislike it, I honestly always intended for our names to be apart of the logo it threw me off when I noticed it wasn't and when I added it it felt more consistent but you think it makes us look ego searching I can just move it back that's no problem.
> 
> It's just my personal take on it. Technically, by next week I likely won't even notice or care that it's there in the sub-menus. And again, it's such a trivial detail that it's not worth spending more time & effort on it.
> 
> > This is probably what caused the confusion when I said "tiny header" you may have thought I meant this signature handle because it wasn't apart of the logo but in my head it was that's why I did the change
> 
> Yes, the signature handle (which was not part of the logo function at the time) is what came to my mind when I read "tiny header" in your reply.

No worries it was my understanding that was wrong I mis-remembered it being apart of the logo and I was just trying to make my recollection/memory accurate haha. But I can change it back I honestly don't care that much myself.

Reply:
Fixed in commit: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/pull/214/commits/e49f95f4e4b4dee6d4aa06f9aefd70f0d8ae2dc2